composer.json

```diff
@@ -20,6 +20,10 @@
         "symfony/filesystem": "~2.5",
         "raulfraile/distill": "~0.9,!=0.9.3,!=0.9.4"
     },
+    "require-dev": {
+        "kherge/box": "~2.5",
```

Comment:
IMO, box should be installed separately (downloading their phar). Otherwise, you will get a bunch of unrelated deps in the phar (Box itself), and it makes things harder to test, as the phar should actually be built without dev deps (I hope the phar distributed for releases does not install dev dependencies in the phar)